manager/controllers/app/moduleinstance.go

```diff
@@ -134,18 +135,20 @@ func (m *ModuleManager) selectReadModule(item modules.DataInfo, appContext *app.
 	m.Log.Info("Select read path for " + item.Context.DataSetID)
 
 	// Read policies for data that is processed in the workload geography
-	var readActions []*pb.EnforcementAction
+	var readActions []*openapiclientmodels.ResultItem
 	var err error
-	readActions, err = LookupPolicyDecisions(item.Context.DataSetID, m.PolicyManager, appContext,
-		&pb.AccessOperation{Type: pb.AccessOperation_READ, Destination: m.WorkloadGeography})
+	reqAction := new(openapiclientmodels.PolicyManagerRequestAction)
+	reqAction.SetActionType(openapiclientmodels.READ)
+	reqAction.SetDestination(m.WorkloadGeography)
+	readActions, err = LookupPolicyDecisions(item.Context.DataSetID, m.PolicyManager, appContext, reqAction)
 	if err != nil {
 		return nil, err
 	}
 	// select a read module that supports user interface requirements
 	// actions are not checked since they are not necessarily done by the read module
 	readSelector := &modules.Selector{Capability: app.Read,
 		Destination:  &item.Context.Requirements.Interface,
-		Actions:      []*pb.EnforcementAction{},
+		Actions:      []*openapiclientmodels.ResultItem{},
```

Comment:
We probably need to change the base taxonomy so `ResultItem` would have another name with more context. Like PolicyManagerResultItem

Reply:
Same as above. We can make this change in the refactoring PR once the end to end flow is tested.

Reply:
Sure